buildAndReleaseTask/index.ts

```diff
@@ -33,7 +34,7 @@ async function run() {
         }
 
         if (mode == "Single") {
-            await updateVersion("./", pathToVersionJson, semverVersion, updateNuspecFiles, updateBuildNumber, addCiLabel, includeParentProps, false, failOnTagVersionMismatch);
+            await updateVersion("./", pathToVersionJson, semverVersion, updateNuspecFiles, updateBuildNumber, addCiLabel, includeParentProps, false, failOnTagVersionMismatch, splitFileVersion);
```

Comment:
Would it make sense to only apply splitFileVersion in case the build number is higher than UInt16.MaxValue? If we implement it like this, it would actually be a breaking change, right?

Reply:
If its done dynamically that could lead to duplicate version numbers 🤔 
`3.2.1.0001` 9998 builds later `3.2.1.9999` 2 builds later `3.2.1.0000` again

Its quite unlikely and can anyways only be prevented when using the `splitFileVersion` from the first build on so 🤷‍♀️ I have no real preference ^^

`splitFileVersion` could also be false per default but that's definitely not ideal.

Reply:
Hmm..true. I think I'd still go with it, since it potentially breaks others otherwise (as we're only using `major` and `minor` of the version). Unless you'd have a better idea, I'd be totally open to that too :D